### PR TITLE
Use new schema. Renamed mounted column to active.

### PR DESCRIPTION
**Pull request by james-s-willis**

### CODE REVIEW

File: alpenhorn/client.py
Comment (ketiltrout) [inline alpenhorn/client.py:695]: You should also rename the `mount` command below.

Also, this is a CLI change.  Do we want to have legacy aliases `mounted`/`mount`?

**Review by ketiltrout — CHANGES_REQUESTED** [review]

File: alpenhorn/client.py
Comment (james-s-willis) [reply alpenhorn/client.py:695]: This `mount` command:
```
@cli.command()
@click.argument("name")
@click.option("--path", help="Root path for this node", type=str, default=None)
@click.option("--user", help="username to access this node.", type=str, default=None)
@click.option(
    "--address", help="address for remote access to this node.", type=str, default=None
)
@click.option(
    "--hostname",
    help="hostname running the alpenhornd instance for this node (set to this hostname by default).",
    type=str,                                                                                            
    default=None,
)
def mount(name, path, user, address, hostname):
    """Interactive routine for mounting a storage node located at ROOT."""

    # We need to write to the database.
    db.connect(read_write=True)

    try:
        node = di.StorageNode.get(name=name)
    except pw.DoesNotExist:
        print('Storage node "%s" does not exist. I quit.' % name)

    if node.active:
        print('Node "%s" is already active.' % name)
        return
```

Comment (james-s-willis) [reply alpenhorn/client.py:695]: Yeah sure, I can add `mount` and `mounted` commands that call `active`.

Comment (ketiltrout) [reply alpenhorn/client.py:695]: Yeah that mount command.  It should be renamed to `activate`.  I realise it _does_ actually try to mount a volume if necessary but it's real job is to activate the node in the DB.  (Then point `mount` at `activate`, not `active`).

Comment (ketiltrout) [reply alpenhorn/client.py:695]: Now that I think about it, there's probably an `unmount` command as well that should get changed to `deactivate`.

Comment (james-s-willis) [reply alpenhorn/client.py:695]: @ketiltrout I've renamed `unmount` -> `deactivate` and kept an alias. I also noticed `unmount/mount_transport`, these functions actually call `mount/umount` commands so I decided to leave them as is.

**Review by ketiltrout — APPROVED** [review]
This looks fine.  It's pretty trivial, all told.

We should wait for the chimedb_di merge before merging this.

**Pull request merged**
